src/site/markdown/server/manage-webadmin.md

```diff
@@ -337,6 +338,23 @@ Response codes:
 
  - 200: The user name list was successfully retrieved
 
+### Retrieving a user allowed From headers list
```

Comment:
the title seems not very clear to me, maybe: Retrieving the list of allowed `From` headers for a given user